src/sctools/metrics/gatherer.py

```diff
@@ -32,16 +32,16 @@ def extract_metrics(self, mode: str='rb') -> None:
             # write the header
             cell_metrics_output.write_header(vars(CellBarcodeMetrics()))
 
-            # break up the bam file into sub-iterators over cells
+            # break up the bam file into sub-iterators over genes
```

Comment:
These three changes here are wrong, I think... 

It should indeed be cell -> molecule -> gene

Reply:
oh did i change cell metrics gatherer instead of gene -_-.  thanks making the change